Lombiq.BaseTheme/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Why is this file here? It shouldn't have to be since we use a root _.eslintrc.js_ in OSOCE. Same for _.stylelintrc.js_.

Reply:
It was generated automatically. So it it shouldn't be here, then should I add it to the .gitignore?

Reply:
@DAud-IcI I forgot that this question is still open.

Reply:
but what is your answer?

Reply:
@0liver ping

Reply:
nvm, it's removed now